### ISSUE TITLE
TypeLoadException in FakeBandClientManager with latest Band Nuget Package 1.3.20217

### ISSUE THREAD

**Issue by hummigbird1** [BandOnTheRun/fake-band#14]
When using the latest Microsof.Band Nuget Package 1.3.20217 
there is a TypeLoadException when using FakeBandClientManager:

An exception of type 'System.TypeLoadException' occurred in BandApp1.exe but was not handled in user code

Additional information: Method 'GetBandsAsync' in type 'FakeBand.Fakes.FakeBandClientManager' from assembly 'FakeBand, Version=0.0.8.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.

This problem does not occur with earlier versions (<= 1.3.20115) of the Microsoft.Band Nuget Package.

Adding the following Code to the App Constructor triggers the TypeLOadException on Debugging Startup (x86 - Local Machine)

```
FakeBand.Fakes.FakeBandClientManager.Configure(
                new FakeBand.Fakes.FakeBandClientManagerOptions() {
                    Bands = new List<IBandInfo>() {
                        new FakeBand.Fakes.FakeBandInfo(BandConnectionType.Bluetooth, "Fake Band 1", FakeBand.Fakes.BandVersion.BandTwo)
                    }
                });
```

**Comment by peted70** [comment]
I am having trouble replicating this issue - can you give some further info on your environment; visual studio version, project type, etc. ?

**Comment by peted70** [comment]
Didn't mean to close the issue

**Comment by bgtsoft** [comment]
Hi I am getting this exact same issue.
I was trying to use FakeBand in a new project created with Template 10 Hamburger.
This is a vanilla project, all I did was update the Template 10 package to the latest one and then Add The Microsoft Band package and the Fake Band package.
I then copy/pasted in your getting started code, as soon as you add the 'FakeBandClientManager.Configure' code the project builds but will not run due to the error listed above. To make sure it wasn't just not playing nice with Template 10 I created a brand new blank app and performed the same steps, again I got that same error.
Finally I tried removing the Microsoft Band NuGet package and just went with the one installed by FakeBand and this time it ran properly.
Seems that FakeBand doesn't want to work with the latest Microsoft Band release.
Dependency >= 1.3.11121
Version installed through NuGet 1.3.20307
VS 2015 Community edition
Regards
Brian

**Comment by hummigbird1** [comment]
Environment is Windows 10 
Visual Studio 2015 Enterprise 

Please find attached the Solution / Project to reproduce the issue on my machine:
[BandApp1.zip](https://github.com/BandOnTheRun/fake-band/files/172648/BandApp1.zip)

Just open the Solution and start Debugging (Solution Configuration "Debug" in Platform "x86") 
Hope this helps.

**Comment by peted70** [comment]
I have updated master to be compatible with the latest band sdk 1.3.20307 so this should be resolved with v0.0.9
